src/pages/namjanghyun/Main/Main.js

```diff
@@ -1,7 +1,37 @@
-import React from "react";
+import React, { useState, useEffect } from "react";
+import "./Main.scss";
+import Feeds from "./components/Feeds";
+import Aside from "./components/Aside";
+import Nav from "../../../components/Nav/Nav";
 
 function MainNam() {
-  return <h1>Hello world</h1>;
+  const [feedDataList, setFeedDataList] = useState([]);
+
+  useEffect(() => {
+    fetch("./data/feedData.json")
```

Comment:
```suggestion
    fetch("/data/feedData.json")
```
public 하위의 절대경로이기 때문에 . 안찍어도 됨!